deploy/teardown_resources.sh

```diff
@@ -7,31 +7,60 @@ set -euxo pipefail
 set_namespace $CONJUR_NAMESPACE_NAME
 
 configure_cli_pod
+
+# Helm Chart clean-up
+rm -f conjur.pem
+
 if [ "${DEV}" = "false" ]; then
-  $cli_with_timeout "exec $(get_conjur_cli_pod_name) -- conjur variable values add secrets/test_secret \"supersecret\""
+  pushd ../../../secrets-provider/ci
+else
+  pushd ../secrets-provider/ci
+fi
+
+  find . -type f ! -name '*template.yaml' -delete
+popd
```

Comment:
something's off with the indentation here

Reply:
purposefully because if you look above it is part of what the pushd` in the if statement. Now that I look at it, It should actually be indented 2 more spaces because it is inside the `pushd`
For example:
```
  pushd ../../../secrets-provider/ci
    find . -type f ! -name '*template.yaml' -delete
  popd
```